pages/locations.md

```diff
@@ -0,0 +1,76 @@
+---
+description: Locations
+draft:       false
+layout:      locations.hbs
+title:       locations
```

Comment:
We should use title case for the page title, so "Locations".

Reply:
Not that quickly, the title is going to change, like I said in the description. The page title is not suited to be used at permalink.